librz/bin/format/elf/elf.h

```diff
@@ -319,6 +316,14 @@ RZ_OWN RzVector *Elf_(rz_bin_elf_new_sections)(RZ_NONNULL ELFOBJ *bin);
 RZ_OWN char *Elf_(rz_bin_elf_section_type_to_string)(ut64 type);
 bool Elf_(rz_bin_elf_has_sections)(RZ_NONNULL ELFOBJ *bin);
 
+// elf_strtab
+
+RZ_OWN RzBinElfStrtab *Elf_(rz_bin_elf_new_strtab)(RZ_NONNULL ELFOBJ *bin, ut64 offset, ut64 size);
+RZ_OWN char *Elf_(rz_bin_elf_strtab_dup)(RZ_NONNULL RzBinElfStrtab *strtab, ut64 index);
+bool Elf_(rz_bin_elf_strtab_cpy)(RZ_NONNULL RzBinElfStrtab *strtab, char *dst, ut64 index);
```

Comment:
I would rename this to `rz_bin_elf_strtab_get()` and annotate dst with `RZ_OUT`. `cpy` to me suggests it clones the entire strtab object.
I think it could also be unified with `rz_bin_elf_strtab_dup` if it strdups whenever `dst == NULL`.

Reply:
rz_bin_elf_strtab_get is a better name. But I am against the idea of merging the 2 functions.

I don't see any clean interface.

Reply:
In that case I would call them `_get` and `_get_dup`. That in combination with the types of the functions should be explanatory enough.